EIPS/eip-7210.md

```diff
@@ -0,0 +1,165 @@
+---
+eip: 7210
```

Comment:
```suggestion
eip: 7208
```
Manually assigning EIP number 7208.  Please also rename the files.

This PR was created 35 minutes after #7207 and #7208 and #7209 are missing.  EIP number sniping and gaming is not allowed

Reply:
oh, man hhahaha... 7210 would have been the perfect number for this standardization :sweat_smile: Thanks for reviewing the PR so quickly.

What's most important is the interface, looking forward to discussing how to standardize it.